site/content/en/docs/Concepts/workflows.md

```diff
@@ -159,6 +159,30 @@ stop the workflow. What happens next depends on the actual workflow engine. For
 engine, any currently executing tasks will halt, any pending tasks will be removed from the work queue and no further
 tasks will be started for that workflow instance.
 
+### `output`
+
+You can obtain the output (if any) of a completed workflow run using the output command and supplying the workflow run
+id. Typically, this is useful for locating the files produced by a workflow, although the actual output generated depends on the workflow
+specification and engine. 
+
+The following is an example of output generated by the Cromwell engine showing the id of the run and the S3 paths out the
+workflow outputs:
+
```

Comment:
Is something missing here? I don't see "an example of output generated by the Cromwell engine".